spec/controllers/api/v1/customer_random_controller_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+RSpec.describe Api::V1::Customers::RandomController, type: :controller do
+  render_views
+
+  describe "returns random customer" do
+    it 'returns a single customer' do
+      customers = create_list(:customer, 3)
+
+      get :show, params: {format: :json}
+      result = JSON.parse(response.body)
+
+      expect(result["id"]).to_not be_nil
```

Comment:
Very good. I hate not being more specific, but since it's random we really can't be.